app/models/post.rb

```diff
@@ -0,0 +1,17 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :likes
+  has_many :comments
+
+  def recent_comments(limit = 5)
+    comments.order(created_at: :desc).limit(limit)
+  end
+
+  def update_comments_counter
+    update(comment_counter: comments.count)
+  end
+
+  def update_likes_counter
```

Comment:
- [ ] This method should be in the `like.rb` model as mentioned in the project requirements. Please, move it there.

[OPTIONAL]
- Just like I mentioned before, you can use rails active callbacks i.e:
`after_save :update_likes_counter`